data/grc/Arnott_1996_Athenaeus/metadata.txt

```diff
@@ -0,0 +1,33 @@
+publicationType : "book"
+publicationTitle : "Alexis. The fragments. A commentary"
+publicationDate : "1996"
+publicationPlace : "Cambridge"
+publicationSeries : ""
+publicationNum: ""
+publicationStemmaNum: "3"
+publicationAuthors : "Arnott, W. Geoffrey"
+publicationPage : ""
+publicationLink : ""
+workTitle : "Δειπνοσοφισταί/Deipnosophistae"
+workViaf : "185515014/187657475"
+workOrigDate : "200?"
+workOrigPlace : "Rome?"
+workAuthor : "Ἀθήναιος/Athenaeus/Athénée"
+workAuthorViaf : "100169636/54186610"
```

Comment:
We do not have strict guidelines regarding redundant/not deduplicated identifiers. I'd say repeat field if several values are necessary (i.e., several authors) or choose the most relevant if not.

Reply:
VIAF can be a mess (I know you know: #34). It is not always clear why we should choose one ID over another.

In this case, the author name is also a problem. It may seem best not to use only a non-Latin script as an author's name. But then… is English the working language of this project? When VIAF has separate entries, the one from the BNF or Sudoc tends to be isolated.

For authors and works in a multilingual perspective, Wikidata has become a nice way to reconcile various IDs. Here the Wikidata author [Q294923](https://www.wikidata.org/wiki/Q294923) page lists 8 different VIAF numbers, and points to the work [Q1244416](https://www.wikidata.org/wiki/Q1244416).

I don't really know what to do. Find a more or less arbitrary criterion and pick one or two VIAF IDs? Select the languages I find releveant?

Reply:
Hmmm, Wikidata is actually a good point. Would it be more consistent than VIAF in the end ? @GusRiva , your opinion ?

Reply:
As for language, we went to english for places, but I would go to 'text language' for the works, authors, etc.
So, for instance, «Arnaut Daniel, _lo ferm voler_», and not «Arnold Danny, _the strong want_». I guess, for Greek works, it means Greek alphabet (and cuneiform for Sumerian texts if we were to include tablets ?).

Reply:
I will include at least English and the original language. Guidelines can still evolve, but any usable information is better than no information – and the structured info carried by the IDs seems the core of the metadata.